locales/ro/translation.json

```diff
@@ -0,0 +1,258 @@
+{
+  "tags-toolbar": {
+    "new-tag": "Eticheta nouă",
+    "tag-statistics": "Statistici etichete",
+    "assign-last-tag": "Atribuie ultima etichetă",
+    "configure-translations": "Configurează traducerile",
+    "configure-dicts": "Configurează dictionarele",
+    "show-parallel-translations": "Compară traducerile",
+    "compare": "Compară",
+    "context": "Context"
+  },
+  "menu": {
+    "book": "Carte",
+    "tags": "Etichete",
+    "search": "Caută",
+    "options": "Optiuni",
+    "export": "Export ca document text",
+    "show-module-info": "Arată info modul",
+    "fullscreen": "Comută pe întregul ecran",
+    "exit-fullscreen": "Ieși din modul ecran-întreg",
+    "text-size": "Ajustează dimensiunea font-ului pentru verset",
+    "text-size-decrease": "Micșorează dimensiunea fontu-lui",
+    "text-size-reset": "Resetează dimensiunea fontu-lui la implicit",
+    "text-size-increase": "Mărește dimensiunea fontu-lui"
+  },
+  "search-menu": {
+    "search": "Căutare",
+    "case-sensitive": "Sensibil la majuscule",
+    "exact-phrase": "Frază exactă",
+    "multi-word": "Cuvinte multiple",
+    "strongs": "Index de concordanță",
+    "extended-verse-boundaries": "Limite extinse pentru verset (Două versete)"
+  },
+  "module-assistant": {
+    "bible-header": "Configurează traducerile biblice",
+    "dict-header": "Configurează dicționarele",
+    "module-type-bible": "Traduceri biblice",
+    "module-type-dict": "Dicționare",
+    "internet-usage-note": "<b>NOTĂ IMPORTANTĂ</b><br/><br/>{{module_type}} sunt instalate folosind conexiunea ta la internet.<br/>Dacă vă aflați într-o țară în care creștinii sunt persecutați, <br/> ar trebui să <b> NU </b> folosiți acest software din cauza riscului de detectare!",
+    "add-translations": "Adaugă traduceri biblice",
+    "remove-translations": "Șterge traduceri biblice",
+    "add-dicts": "Adaugă dicționare",
+    "remove-dicts": "Șterge dicționare",
+    "repositories": "Depozite",
+    "repo-data-last-updated": "Datele din depozit au fost actualizate ultima oară la {{date}}.",
+    "update-now": "Actualizează acum",
+    "languages": "Limbi",
+    "translations": "Traduceri",
+    "removal": "Ștergere",
+    "module-feature-filter": "Filtru de funcții",
+    "repo-selection-info-text":"Aplicația Ezra Bible funcționează cu {{module_type}} furnizate de <a class='external' href='http://www.crosswire.org/sword'> proiectul SWORD </a> și <a class = 'external' href='http://www.crosswire.org '> CrossWire Bible Society </a>. <br/> <br/> Mai jos ai lista depozitelor SWORD publicate de CrossWire. Un depozit este o locație de stocare pe internet care conține un set de {{module_type}}. Lângă fiecare depozit, vezi numărul total de {{module_type}} disponibile din depozitul respectiv. <br/> <br/> Pentru a instala un modul, selectează cel puțin un depozit din lista de mai jos. Ai putea începe cu depozitele <i> CrossWire </i> și <i> eBible.org </i>.",
```

Comment:
I would suggest to keep the product name "Ezra Bible App" unmodified wherever it is used.

Reply:
ok, i will do the change